Allow multiple IPs in docker IP env variable

SoLoR1 opened the pull request:
#59

simonmicro commented:
And this works - interesting... What do you think @Matthew-Beckett ? Does the server still work?

SoLoR1 commented:
Well it shouldnt mess with anything. What i did was, changed listen_ip to array and split IP enviroment variabile by spaces (If its only one IP or 0.0.0.0 it arrays of one). It takes first IP in array for pykms_Client pykms_Server and sqlite and in case there are more IPs it adds connect and other arguments to pykms_Server with other ips that are provided. But yes maybe there is more "slick" way of doing it, but this works and it shouldnt break anything for anyone.

It only affects docker container and nothing else. In case i feel like doing more python learning im gonna look in to pykms_Server network code and try to do all arguments and listening on IPs/interfaces more intuitively, but considering im kinda begginner and thats a lot of code spread around multiple modules....
